core/src/main/java/io/fabric8/maven/core/util/FileUtil.java

```diff
@@ -28,6 +30,14 @@
  */
 public class FileUtil {
 
+    public static File createTempDirectory() {
+        try {
+            return Files.createTempDirectory("fabric8").toFile();
```

Comment:
It would be nice if we pass directory name as a parameter ;-)

Reply:
Currentñy this is not the directory, it is the prefix, Java creates a random directory in temporal space with `fabric8` prefix. So I think that we should not parametrize this method to maintain safely the temp directory to not populating with a different prefix.

Reply:
Oh, are you creating file in `/tmp` directory?